compiler-core/src/warning.rs

```diff
@@ -437,6 +437,22 @@ expression.",
                     }),
                 },
 
+                type_::Warning::UnusedImportedModuleAlias { location, .. } => Diagnostic {
+                    title: "Unused alias for imported module".into(),
+                    text: "".into(),
+                    hint: Some("You can safely remove it.".into()),
```

Comment:
This might not be true in the case where there is another import using the default name for the module.

Reply:
I'm not sure what you mean by this.
Is this the situation you're talking about?
```gleam
import gleam/io
import two.{four} as _one
import one.{three} as two

pub fn main() {
  io.debug(three)
  io.debug(four)
}
```
I ran this and I got:
```
warning: Unused alias for imported module
  ┌─ <path>/src/dojo.gleam:3:20
  │
3 │ import one.{three} as two
  │                    ^^^^^^ This alias for the imported module is never used.

Hint: You can safely remove it.
```
Which is correct!

Reply:
```gleam
import a/one
import a/one as two
```

Here it'll say you can safely remove `as two`, but that is not the case as it will cause a compile error.

Reply:
It's fixed!

Reply:
Sorry, how has it been fixed? It still says you can remove it.

Reply:
I think there is a misunderstanding. I just tested this code
```gleam
import a/one
import b/one as two 

pub fn main() {
  todo
}
```
The only warning I get is for modules themselves not being used which is not incorrect. It does not mention the alias at all:
```
warning: Unused imported module
  ┌─ src/dojo.gleam:1:1
  │
1 │ import a/one
  │ ^^^^^^^^^^^^ This imported module is never used.

Hint: You can safely remove it.

warning: Unused imported module
  ┌─ src/dojo.gleam:2:1
  │
2 │ import b/one as two
  │ ^^^^^^^^^^^^^^^^^^^ This imported module is never used.

Hint: You can safely remove it.
   Compiled in 0.36s
```

Reply:
I should add that this won't emit a warning, as It shouldn't:
```gleam
import gleam/io
import a/one.{three}
import b/one.{four} as two

pub fn main() {
  io.debug(three)
  io.debug(four)
}
```